<?xml version="1.0" encoding="UTF-8"?><Posts><Post><PostId>1788245</PostId><Url>http://stackoverflow.com/questions/1788245</Url><Stack><Type>TemplateSyntaxError</Type><Message>&lt;</Message><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\tools\dev_appserver.py</File><Line>2978</Line><Method>_HandleRequest</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\tools\dev_appserver.py</File><Line>411</Line><Method>Dispatch</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\tools\dev_appserver.py</File><Line>2244</Line><Method>Dispatch</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\tools\dev_appserver.py</File><Line>2162</Line><Method>ExecuteCGI</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\tools\dev_appserver.py</File><Line>2058</Line><Method>ExecuteOrImportScript</Method></Frame><Frame><File>C:\Users\Ben\Development\workspace\ringbot\src\main.py</File><Line>52</Line><Method>module</Method></Frame><Frame><File>C:\Users\Ben\Development\workspace\ringbot\src\main.py</File><Line>49</Line><Method>main</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\webapp\util.py</File><Line>97</Line><Method>run_wsgi_app</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\webapp\util.py</File><Line>115</Line><Method>run_bare_wsgi_app</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\core\handlers\wsgi.py</File><Line>241</Line><Method>__call__</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\core\handlers\base.py</File><Line>134</Line><Method>get_response</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\core\handlers\base.py</File><Line>154</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\views\debug.py</File><Line>40</Line><Method>technical_500_response</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\views\debug.py</File><Line>114</Line><Method>get_traceback_html</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\template\__init__.py</File><Line>178</Line><Method>render</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\template\__init__.py</File><Line>779</Line><Method>render</Method></Frame><Frame><File>C:\Python25\lib\site-packages\django\template\debug.py</File><Line>81</Line><Method>render_node</Method></Frame></Stack></Post><Post><PostId>2986258</PostId><Url>http://stackoverflow.com/questions/2986258</Url><Stack><Type>TemplateSyntaxError</Type><Message>Template u'common.html' cannot be extended, because it doesn't exist</Message><Frame><File>C:\code\kwbudget\edit_budget.py</File><Line>34</Line><Method>get</Method></Frame><Frame><File>C:\code\kwbudget\django\template\__init__.py</File><Line>165</Line><Method>render</Method></Frame><Frame><File>C:\code\kwbudget\django\template\__init__.py</File><Line>784</Line><Method>render</Method></Frame><Frame><File>C:\code\kwbudget\django\template\__init__.py</File><Line>797</Line><Method>render_node</Method></Frame><Frame><File>C:\code\kwbudget\django\template\loader_tags.py</File><Line>71</Line><Method>render</Method></Frame><Frame><File>C:\code\kwbudget\django\template\loader_tags.py</File><Line>66</Line><Method>get_parent</Method></Frame></Stack></Post><Post><PostId>5078124</PostId><Url>http://stackoverflow.com/questions/5078124</Url><Stack><Type>TemplateSyntaxError</Type><Message>Could not parse the remainder: :'M j, H:i'</Message><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>515</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/.../1.348530548823349202/main.py</File><Line>22</Line><Method>get</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py</File><Line>155</Line><Method>render</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py</File><Line>183</Line><Method>load</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader.py</File><Line>80</Line><Method>get_template</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader.py</File><Line>88</Line><Method>get_template_from_string</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>158</Line><Method>__init__</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>174</Line><Method>compile_string</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/defaulttags.py</File><Line>544</Line><Method>do_for</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>254</Line><Method>parse</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>338</Line><Method>compile_filter</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>558</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>5584411</PostId><Url>http://stackoverflow.com/questions/5584411</Url><Stack><Type>TemplateSyntaxError</Type><Message>Caught ImproperlyConfigured while rendering: 'django.db.backends.postgresql' isn't an available database backend.</Message><Frame><File>/usr/lib/python2.6/dist-packages/mod_python/importer.py</File><Line>1537</Line><Method>HandlerDispatch</Method></Frame><Frame><File>/usr/lib/python2.6/dist-packages/mod_python/importer.py</File><Line>1229</Line><Method>_process_target</Method></Frame><Frame><File>/usr/lib/python2.6/dist-packages/mod_python/importer.py</File><Line>1128</Line><Method>_execute_target</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/core/handlers/modpython.py</File><Line>228</Line><Method>handler</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/core/handlers/modpython.py</File><Line>201</Line><Method>__call__</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/core/handlers/base.py</File><Line>141</Line><Method>get_response</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/core/handlers/base.py</File><Line>165</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/views/debug.py</File><Line>58</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/views/debug.py</File><Line>137</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/__init__.py</File><Line>173</Line><Method>render</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/__init__.py</File><Line>167</Line><Method>_render</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/__init__.py</File><Line>796</Line><Method>render</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/debug.py</File><Line>72</Line><Method>render_node</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/debug.py</File><Line>89</Line><Method>render</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/__init__.py</File><Line>579</Line><Method>resolve</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/template/defaultfilters.py</File><Line>697</Line><Method>date</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/dateformat.py</File><Line>281</Line><Method>format</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/dateformat.py</File><Line>187</Line><Method>r</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/encoding.py</File><Line>66</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/translation/__init__.py</File><Line>55</Line><Method>ugettext</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/functional.py</File><Line>55</Line><Method>_curried</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/translation/__init__.py</File><Line>36</Line><Method>delayed_loader</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/translation/trans_real.py</File><Line>276</Line><Method>ugettext</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/translation/trans_real.py</File><Line>266</Line><Method>do_translate</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/translation/trans_real.py</File><Line>176</Line><Method>translation</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/translation/trans_real.py</File><Line>159</Line><Method>_fetch</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/contrib/admin/__init__.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/contrib/admin/helpers.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/forms/__init__.py</File><Line>17</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/forms/models.py</File><Line>6</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/db/__init__.py</File><Line>77</Line><Method>module</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/db/utils.py</File><Line>91</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/lib/pymodules/python2.6/django/db/utils.py</File><Line>49</Line><Method>load_backend</Method></Frame></Stack></Post><Post><PostId>5677601</PostId><Url>http://stackoverflow.com/questions/5677601</Url><Stack><Type>TemplateSyntaxError</Type><Message>'if' statement improperly formatted</Message><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\webapp\__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>C:\Store house\gae\community\src\community.py</File><Line>24</Line><Method>get</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\webapp\template.py</File><Line>72</Line><Method>render</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\google\appengine\ext\webapp\template.py</File><Line>100</Line><Method>load</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\loader.py</File><Line>80</Line><Method>get_template</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\loader.py</File><Line>88</Line><Method>get_template_from_string</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\__init__.py</File><Line>158</Line><Method>__init__</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\__init__.py</File><Line>174</Line><Method>compile_string</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\loader_tags.py</File><Line>154</Line><Method>do_extends</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\loader_tags.py</File><Line>132</Line><Method>do_block</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>C:\Program Files\Google\google_appengine\lib\django_0_96\django\template\defaulttags.py</File><Line>655</Line><Method>do_if</Method></Frame></Stack></Post><Post><PostId>5688216</PostId><Url>http://stackoverflow.com/questions/5688216</Url><Stack><Type>TemplateSyntaxError</Type><Message>Unclosed tags: endblock, endblock bodycontent</Message><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/hoosheer/4.349803081119561985/hoosheer_main.py</File><Line>34</Line><Method>get</Method></Frame><Frame><File>/base/data/home/apps/hoosheer/4.349803081119561985/foursquareMethods.py</File><Line>40</Line><Method>foursquareDetails</Method></Frame><Frame><File>/base/data/home/apps/hoosheer/4.349803081119561985/foursquareMethods.py</File><Line>67</Line><Method>doRender</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py</File><Line>72</Line><Method>render</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py</File><Line>100</Line><Method>load</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader.py</File><Line>80</Line><Method>get_template</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader.py</File><Line>88</Line><Method>get_template_from_string</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>158</Line><Method>__init__</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>174</Line><Method>compile_string</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader_tags.py</File><Line>154</Line><Method>do_extends</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader_tags.py</File><Line>132</Line><Method>do_block</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>280</Line><Method>parse</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>318</Line><Method>unclosed_block_tag</Method></Frame></Stack></Post><Post><PostId>5809647</PostId><Url>http://stackoverflow.com/questions/5809647</Url><Stack><Type>TemplateSyntaxError</Type><Message>Could not parse the remainder: ()</Message><Frame><File>/opt/google_appengine/google/appengine/ext/webapp/__init__.py</File><Line>634</Line><Method>__call__</Method></Frame><Frame><File>/home/thomas/workspace/movieMashup/src/main.py</File><Line>71</Line><Method>get</Method></Frame><Frame><File>/opt/google_appengine/google/appengine/ext/webapp/template.py</File><Line>72</Line><Method>render</Method></Frame><Frame><File>/opt/google_appengine/google/appengine/ext/webapp/template.py</File><Line>100</Line><Method>load</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/loader.py</File><Line>80</Line><Method>get_template</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/loader.py</File><Line>88</Line><Method>get_template_from_string</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>158</Line><Method>__init__</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>174</Line><Method>compile_string</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/loader_tags.py</File><Line>154</Line><Method>do_extends</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/loader_tags.py</File><Line>132</Line><Method>do_block</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>273</Line><Method>parse</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/defaulttags.py</File><Line>542</Line><Method>do_for</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>338</Line><Method>compile_filter</Method></Frame><Frame><File>/opt/google_appengine/lib/django_0_96/django/template/__init__.py</File><Line>558</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>8648310</PostId><Url>http://stackoverflow.com/questions/8648310</Url><Stack><Type>TemplateSyntaxError</Type><Message>Caught VariableDoesNotExist while rendering: Failed lookup for key [user] in u"'uploaded_photos/Published/2011/December/21/image_5.jpeg'"</Message><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/contrib/staticfiles/handlers.py</File><Line>68</Line><Method>__call__</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/contrib/staticfiles/handlers.py</File><Line>68</Line><Method>__call__</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/core/handlers/base.py</File><Line>111</Line><Method>get_response</Method></Frame><Frame><File>/Users/glosseyvilly/PycharmProjects/first_django_dev/apps/photos/views.py</File><Line>113</Line><Method>search_photos</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/shortcuts/__init__.py</File><Line>44</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/loader.py</File><Line>188</Line><Method>render_to_string</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/loader_tags.py</File><Line>127</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/loader_tags.py</File><Line>64</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/defaulttags.py</File><Line>311</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/defaulttags.py</File><Line>227</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>915</Line><Method>render</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>653</Line><Method>resolve</Method></Frame><Frame><File>/Users/glosseyvilly/Desktop/web_dev/ccdev/lib/python2.6/site-packages/django/template/base.py</File><Line>692</Line><Method>_resolve_lookup</Method></Frame></Stack></Post><Post><PostId>9468402</PostId><Url>http://stackoverflow.com/questions/9468402</Url><Stack><Type>TemplateSyntaxError</Type><Message>Caught ImproperlyConfigured while rendering: Error loading MySQLdb module: dlopen(/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/MySQL_python-1.2.3-py2.7-macosx-10.4-x86_64.egg/_mysql.so, 2): Library not loaded: libmysqlclient.18.dylib</Message><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/views/debug.py</File><Line>151</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/debug.py</File><Line>90</Line><Method>render</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/base.py</File><Line>536</Line><Method>resolve</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/template/defaultfilters.py</File><Line>695</Line><Method>date</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>285</Line><Method>format</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>191</Line><Method>r</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/encoding.py</File><Line>71</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/translation/__init__.py</File><Line>81</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>286</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>276</Line><Method>do_translate</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>185</Line><Method>translation</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>162</Line><Method>_fetch</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/contrib/admin/__init__.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/contrib/admin/helpers.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/contrib/admin/util.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/db/__init__.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/db/utils.py</File><Line>93</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/db/utils.py</File><Line>33</Line><Method>load_backend</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/Cellar/python/2.7.2/lib/python2.7/site-packages/django/db/backends/mysql/base.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>9475626</PostId><Url>http://stackoverflow.com/questions/9475626</Url><Stack><Type>TemplateSyntaxError</Type><Message>Caught ImproperlyConfigured while rendering: Error loading MySQLdb module: No module named MySQLdb</Message><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>151</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>90</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>536</Line><Method>resolve</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/defaultfilters.py</File><Line>695</Line><Method>date</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>285</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>191</Line><Method>r</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/encoding.py</File><Line>71</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/__init__.py</File><Line>81</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>286</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>276</Line><Method>do_translate</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>185</Line><Method>translation</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>162</Line><Method>_fetch</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/__init__.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/helpers.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/util.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/__init__.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>93</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>33</Line><Method>load_backend</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/backends/mysql/base.py</File><Line>14</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>151</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>90</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>536</Line><Method>resolve</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/defaultfilters.py</File><Line>695</Line><Method>date</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>285</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>191</Line><Method>r</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/encoding.py</File><Line>71</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/__init__.py</File><Line>81</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>286</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>276</Line><Method>do_translate</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>185</Line><Method>translation</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>162</Line><Method>_fetch</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/__init__.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/helpers.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/util.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/__init__.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>93</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>33</Line><Method>load_backend</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/backends/mysql/base.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>9475626</PostId><Url>http://stackoverflow.com/questions/9475626</Url><Stack><Type>TemplateSyntaxError</Type><Message>Caught ImproperlyConfigured while rendering: Error loading MySQLdb module: No module named MySQLdb</Message><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>151</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>90</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>536</Line><Method>resolve</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/defaultfilters.py</File><Line>695</Line><Method>date</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>285</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>191</Line><Method>r</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/encoding.py</File><Line>71</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/__init__.py</File><Line>81</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>286</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>276</Line><Method>do_translate</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>185</Line><Method>translation</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>162</Line><Method>_fetch</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/__init__.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/helpers.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/util.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/__init__.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>93</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>33</Line><Method>load_backend</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/backends/mysql/base.py</File><Line>14</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>151</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>90</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>536</Line><Method>resolve</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/defaultfilters.py</File><Line>695</Line><Method>date</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>285</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>191</Line><Method>r</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/encoding.py</File><Line>71</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/__init__.py</File><Line>81</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>286</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>276</Line><Method>do_translate</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>185</Line><Method>translation</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>162</Line><Method>_fetch</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/__init__.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/helpers.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/util.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/__init__.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>93</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>33</Line><Method>load_backend</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/backends/mysql/base.py</File><Line>14</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>12238371</PostId><Url>http://stackoverflow.com/questions/12238371</Url><Stack><Type>TemplateSyntaxError</Type><Message>Caught ImportError while rendering: cannot import name utils</Message><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py</File><Line>272</Line><Method>__call__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>169</Line><Method>get_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>203</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>59</Line><Method>technical_500_response</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/views/debug.py</File><Line>151</Line><Method>get_traceback_html</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>123</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>117</Line><Method>_render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>744</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>73</Line><Method>render_node</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/debug.py</File><Line>90</Line><Method>render</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/base.py</File><Line>536</Line><Method>resolve</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/template/defaultfilters.py</File><Line>695</Line><Method>date</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>285</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>191</Line><Method>r</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/dateformat.py</File><Line>30</Line><Method>format</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/encoding.py</File><Line>71</Line><Method>force_unicode</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/functional.py</File><Line>206</Line><Method>__unicode_cast</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/__init__.py</File><Line>81</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>286</Line><Method>ugettext</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>276</Line><Method>do_translate</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>185</Line><Method>translation</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py</File><Line>162</Line><Method>_fetch</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/__init__.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/helpers.py</File><Line>3</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/contrib/admin/util.py</File><Line>1</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/__init__.py</File><Line>78</Line><Method>module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>93</Line><Method>__getitem__</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/utils.py</File><Line>33</Line><Method>load_backend</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/utils/importlib.py</File><Line>35</Line><Method>import_module</Method></Frame><Frame><File>/usr/local/lib/python2.7/site-packages/django/db/backends/postgresql_psycopg2/base.py</File><Line>9</Line><Method>module</Method></Frame></Stack></Post><Post><PostId>13409460</PostId><Url>http://stackoverflow.com/questions/13409460</Url><Stack><Type>TemplateSyntaxError</Type><Message>Could not parse the remainder: {*/\nbody {\n   background: [[ColorPalette::Background]];\n color: [[ColorPalette::Foreground]]</Message><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/_webapp25.py</File><Line>710</Line><Method>__call__</Method></Frame><Frame><File>/base/data/home/apps/s~americocunhajr/1.363201743143209914/main.py</File><Line>16</Line><Method>get</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py</File><Line>91</Line><Method>render</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py</File><Line>113</Line><Method>_load_user_django</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader.py</File><Line>80</Line><Method>get_template</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader.py</File><Line>88</Line><Method>get_template_from_string</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>158</Line><Method>__init__</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>174</Line><Method>compile_string</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>254</Line><Method>parse</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>338</Line><Method>compile_filter</Method></Frame><Frame><File>/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py</File><Line>558</Line><Method>__init__</Method></Frame></Stack></Post><Post><PostId>15908039</PostId><Url>http://stackoverflow.com/questions/15908039</Url><Stack><Type>TemplateSyntaxError</Type><Message>{% get_metadata %} needs some path information.</Message><Frame><File>/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/wsgiref/handlers.py</File><Line>85</Line><Method>run</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-  packages/django/core/handlers/wsgi.py</File><Line>255</Line><Method>__call__</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>178</Line><Method>get_response</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>224</Line><Method>handle_uncaught_exception</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/utils/decorators.py</File><Line>91</Line><Method>_wrapped_view</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/views/defaults.py</File><Line>41</Line><Method>server_error</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>140</Line><Method>render</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>134</Line><Method>_render</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>830</Line><Method>render</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>844</Line><Method>render_node</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/loader_tags.py</File><Line>124</Line><Method>render</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>134</Line><Method>_render</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>830</Line><Method>render</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/django/template/base.py</File><Line>844</Line><Method>render_node</Method></Frame><Frame><File>/Users/Sreekanth/django1.5/django1.5_ve/lib/python2.7/site-packages/rollyourown/seo/templatetags/seo.py</File><Line>25</Line><Method>render</Method></Frame></Stack></Post><Post><PostId>22026177</PostId><Url>http://stackoverflow.com/questions/22026177</Url><Stack><Type>TemplateSyntaxError</Type><Message>'block' tag with name 'content' appears more than once</Message><Frame><File>/vagrant/webapps/nns/settings/../apps/story/tests.py</File><Line>11</Line><Method>test_root_url_shows_home_page_content</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/test/client.py</File><Line>473</Line><Method>get</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/test/client.py</File><Line>280</Line><Method>get</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/test/client.py</File><Line>444</Line><Method>request</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/core/handlers/base.py</File><Line>152</Line><Method>get_response</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/utils/decorators.py</File><Line>99</Line><Method>_wrapped_view</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/views/defaults.py</File><Line>23</Line><Method>page_not_found</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader.py</File><Line>138</Line><Method>get_template</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader.py</File><Line>127</Line><Method>find_template</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader.py</File><Line>43</Line><Method>__call__</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader.py</File><Line>49</Line><Method>load_template</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader.py</File><Line>149</Line><Method>get_template_from_string</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/base.py</File><Line>125</Line><Method>__init__</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/base.py</File><Line>153</Line><Method>compile_string</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/base.py</File><Line>278</Line><Method>parse</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader_tags.py</File><Line>215</Line><Method>do_extends</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/base.py</File><Line>278</Line><Method>parse</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader_tags.py</File><Line>190</Line><Method>do_block</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/base.py</File><Line>278</Line><Method>parse</Method></Frame><Frame><File>/home/vagrant/django5/local/lib/python2.7/site-packages/django/template/loader_tags.py</File><Line>186</Line><Method>do_block</Method></Frame></Stack></Post><Post><PostId>22598618</PostId><Url>http://stackoverflow.com/questions/22598618</Url><Stack><Type>TemplateSyntaxError</Type><Message>expected token 'end of print statement', got '.'</Message><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1535</Line><Method>__call__</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1529</Line><Method>__call__</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1278</Line><Method>default_dispatcher</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>1102</Line><Method>__call__</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>572</Line><Method>dispatch</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/webapp2-2.5.2/webapp2.py</File><Line>570</Line><Method>dispatch</Method></Frame><Frame><File>/Users/john/src/employee/frontend/views.py</File><Line>38</Line><Method>get</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/jinja2-2.6/jinja2/environment.py</File><Line>719</Line><Method>get_template</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/jinja2-2.6/jinja2/environment.py</File><Line>693</Line><Method>_load_template</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/jinja2-2.6/jinja2/loaders.py</File><Line>127</Line><Method>load</Method></Frame><Frame><File>/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/lib/jinja2-2.6/jinja2/environment.py</File><Line>493</Line><Method>compile</Method></Frame><Frame><File>/Users/john/src/employee/frontend/templates/index.html</File><Line>26</Line><Method>template</Method></Frame></Stack></Post></Posts>